cli/flox/src/commands/services/stop.rs

```diff
@@ -0,0 +1,40 @@
+use anyhow::Result;
+use bpaf::Bpaf;
+use flox_rust_sdk::flox::Flox;
+use flox_rust_sdk::providers::services::stop_services;
+use tracing::instrument;
+
+use crate::commands::{environment_select, EnvironmentSelect};
+use crate::subcommand_metric;
+use crate::utils::message;
+
+#[derive(Bpaf, Debug, Clone)]
+pub struct Stop {
+    #[bpaf(external(environment_select), fallback(Default::default()))]
+    environment: EnvironmentSelect,
+
+    /// Names of the services to stop
+    #[bpaf(positional("name"))]
+    names: Vec<String>,
+}
+
+impl Stop {
+    // TODO: are these nested services->stop?
+    #[instrument(name = "stop", skip_all)]
+    pub async fn handle(self, flox: Flox) -> Result<()> {
+        // TODO: include spaces?
+        subcommand_metric!("services stop");
```

Comment:
I need some input on these from someone that's more familiar with our telemetry setup.

Reply:
From sync with Zach:
- Sentry `instrument` spans are nested correctly so we can just removed the TODO
- No idea about the inhouse `subcommand_metric` one at the moment

Reply:
I got rid of the first `todo`

Reply:
I've changed `subcommand_metric` to double-colon delimited for now.